python/skupper_router/management/skrouter.json

```diff
@@ -618,7 +618,7 @@
                     "required": false
                 },
                 "dataConnectionCount": {
-                    "description": "The number of parallel data connections to carry streaming data between routers. Applies only to interior routers",
+                    "description": "The number of parallel data connections to carry streaming data between routers. Applies only to interior routers. Defaults to 'auto', don't set to 0 (zero), if set to zero, will be forced to 1",
```

Comment:
Not something you changed but maybe this description should be more specific WRT the connection role.  Like "Applies only to connectors with a role of "inter-router"???  Or something like that...